tla-io/src/main/scala/at/forsyte/apalache/io/lir/PrettyWriter.scala

```diff
@@ -32,6 +32,8 @@ class PrettyWriter(
     extends PrettyPrinter with TlaWriter {
   override val defaultIndent: Int = layout.indent
 
+  val defaultNameResolver = (x: String) => x
```

Comment:
This is referred to at some points, but I think it's never used, and the argument to `write` is explicitly defined?